eas-build-pre-install.sh

```diff
@@ -3,5 +3,7 @@
 # If the OS is MacOS, install Java so that the shadow-cljs build succeeds
 if [[ $OSTYPE == "darwin"* ]]; then
   brew install openjdk@11
-  sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk
-fi
\ No newline at end of file
+  sudo ln -sfn /opt/homebrew/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk
```

Comment:
This will if the machine does not have jdk-11 installed by homebrew. Which we probably can assume it has, but anyway, probably good to log something that can give a clue to what is failing.

```suggestion
  sudo ln -sfn /opt/homebrew/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk || { echo 'Homebrew JDK 11 not found?' ; exit 1; }
```